openhcl/openhcl_boot/src/main.rs

```diff
@@ -508,6 +511,18 @@ const fn zeroed<T: FromZeroes>() -> T {
     unsafe { core::mem::MaybeUninit::<T>::zeroed().assume_init() }
 }
 
+/// Get the tsc ref time for hw isolated vms in 100ns
+#[allow(unused_variables)]
+fn get_hw_tsc_time(isolation: IsolationType) -> u64 {
```

Comment:
can we rewrite this as a match statement? we can cfg gate the different isolation types, but that would remove the `unused_variables`